Intersect.Client/Interface/Game/EntityPanel/EntityBox.cs

```diff
@@ -801,7 +801,7 @@ private void UpdateXpBar(float elapsedTime, bool instant = false)
             float targetExpSize;
             var barDirectionSetting = ClientConfiguration.Instance.EntityBarDirections[(int)Vital.VitalCount];
             var barPercentageSetting = Globals.Database.ShowExperienceAsPercentage;
-            var entityExperienceToNextLevel = (float)((Player)MyEntity).GetNextLevelExperience();
+            var entityExperienceToNextLevel = ((Player)MyEntity).GetNextLevelExperience();
```

Comment:
This should be the only line needed for this PR. The rest of the PR is unnecessary.